_data/simple-icons.json

```diff
@@ -4530,7 +4533,7 @@
         {
             "title": "JavaScript",
             "hex": "F7DF1E",
-            "source": "https://github.com/voodootikigod/logo.js",
+            "source": "https://github.com/voodootikigod/logo.js/blob/master/js.png",
```

Comment:
Would be better to use a vector version for this one: https://github.com/voodootikigod/logo.js/blob/1544bdeed6d618a6cfe4f0650d04ab8d9cfa76d9/js.svg